web/templates/reference.html

```diff
@@ -26,21 +26,17 @@ <h3 class="text-center">{{ lang_friendlyname|linebreaksbr }}'s Implementation</h
                 </div>
             </div>
 
-    {% for concept_item in category.concepts %}
-        {% for c in concepts %}
-            {% if c.id == concept_item %}
-            <div class="card-group">
-                <div class="card">
-                    <div class="card-body">
-                        <div class="strong">
-                            {{ c.name|linebreaksbr }}
-                        </div>
+    {% for concept in category.concepts %}
+        <div class="card-group">
+            <div class="card">
+                <div class="card-body">
+                    <div class="strong">
+                        {{ concept.name|linebreaksbr }}
```

Comment:
Shouldn't this also have a default?

Reply:
I think rather than adding default here we should remove it above, if I'm not mistaken we handle missing concepts in the backend and should never need to have a default here.

Reply:
Yeah, it should be in the backend too. I thought you were moving it from backend to frontend. Whichever!

Reply:
```suggestion
                        {{ concept.name | default:"Unsupported or Not Needed" | linebreaksbr }}
```